apis/supervisor/config/v1alpha1/types_oidcclient.go.tmpl

```diff
@@ -75,17 +75,20 @@ type OIDCClientSpec struct {
 
 // OIDCClientStatus is a struct that describes the actual state of an OIDCClient.
 type OIDCClientStatus struct {
-	// Phase summarizes the overall status of the OIDCClient.
+	// phase summarizes the overall status of the OIDCClient.
 	// +kubebuilder:default=Pending
 	// +kubebuilder:validation:Enum=Pending;Ready;Error
 	Phase OIDCClientPhase `json:"phase,omitempty"`
 
-	// Represents the observations of an OIDCClient's current state.
+	// conditions represent the observations of an OIDCClient's current state.
 	// +patchMergeKey=type
 	// +patchStrategy=merge
 	// +listType=map
 	// +listMapKey=type
 	Conditions []Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+
+	// totalClientSecrets is the current number of client secrets that are detected for this OIDCClient.
+	TotalClientSecrets int `json:"totalClientSecrets"`
```

Comment:
Thanks for noticing. All of the printer column stuff got moved to another (new) user story because in the original sequence of stories it was not clear when that implementation effort (and story acceptance effort) should happen. So we'll circle back on that in the near future in a new PR.